server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/Pullrequest.java

```diff
@@ -0,0 +1,64 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.lang.NonNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.base.BaseGitServiceEntity;
+import de.tum.in.www1.hephaestus.codereview.comment.IssueComment;
+import de.tum.in.www1.hephaestus.codereview.repository.Repository;
+import de.tum.in.www1.hephaestus.codereview.user.User;
+import jakarta.persistence.Table;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "pull_request")
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@ToString(callSuper = true)
+public class PullRequest extends BaseGitServiceEntity {
```

Comment:
I think you need to rename the files starting with `PullRequest`, I'm getting an error here or is this on my side?

Reply:
Seems like you are right, Git did not push the rename. For me in VS Code everything works and is correctly renamed.
I can try it in a different environment.